store/src/java/com/zimbra/cs/util/AccountUtil.java

```diff
@@ -922,10 +922,10 @@ public static boolean isDocumentEditingEnabled(Account account) {
         if (isEnabled) {
             Provisioning prov = Provisioning.getInstance();
             try {
+                isEnabled = false;
```

Comment:
And this will work for zimbraFeatureDocumentEditingEnabled and zimbraDocumentServerHost for true too along with all document editor functionality?

Reply:
yes